src/scripts/exclusion_term_expansion/exclusion_term_expansion.py

```diff
@@ -0,0 +1,299 @@
+"""Takes in the full ontology and the exclusions tsv and extracts a simple list of terms from it.
+Outputs two files:
+1. a simple list of terms (config/ONTO_NAME_term_exclusions.txt)
+2. a simple two column file with the term_id and the exclusion reason
+
+# Resources
+- GitHub issue: https://github.com/monarch-initiative/mondo-ingest/issues/22
+
+TODO's
+  - todo: Do we need to worry about edge case where exclusion table is empty? (e.g. we include such a table for all
```

Comment:
`todo: Do we need to worry about edge case where exclusion table is empty? (e.g. we include such a table for all ontologies, but there exists 1 where we don't exclude anything.`
I can add this edge-case handling if you want.

Reply:
yes, lets just handle this case and do nothing.